BoxSelector/index.jsx

```diff
@@ -55,4 +61,6 @@ BoxSelector.propTypes = {
   layout: PropTypes.oneOf(['auto', 'horizontal', 'vertical'])
 }
 
-export default BoxSelector
```

Comment:
Thanks @joaomilho , you are right. `compose` end up as leftover of the initial solution :)